sources/api/bootstrap-containers/src/main.rs

```diff
@@ -299,7 +301,13 @@ where
 
     // Write the environment file needed for the systemd service to have details
     // this specific bootstrap container
-    write_config_files(name, source, &mode, essential)?;
+    write_config_files(
+        name,
+        source,
+        &mode,
+        essential,
+        container_details.command.join(","),
```

Comment:
For overall safety against weird edge cases, like`,` as part of the command - might be good to borrow the `toml_encode` trick and write the vec out as a JSON string. Then `host-ctr` can deserialize from JSON back into an array.